source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.6.0.rst

```diff
@@ -0,0 +1,66 @@
+.. _K8SPXC-1.6.0:
+
+================================================================================
+*Percona Kubernetes Operator for Percona XtraDB Cluster* 1.6.0
+================================================================================
+
+:Date: October 7, 2020
+:Installation: `Installing Percona Kubernetes Operator for Percona XtraDB Cluster <https://www.percona.com/doc/kubernetes-operator-for-pxc/index.html#quickstart-guides>`_
+
+New Features
+================================================================================
+
+* :jirabug:`K8SPXC-416`: :ref:`Support of the proxy-protocol<haproxy-conf-protocol>` in HAProxy
+* :jirabug:`K8SPXC-429`: Possibility to :ref:`restore backups to a new Kubernetes-based environment<backups-restore>`
+* :jirabug:`K8SPXC-144`: Allow :ref:`adding ProxySQL configuration options<proxysql-conf-custom>`
+* :jirabug:`K8SPXC-343`: Helm chart :ref:`officially provided with the Operator`
+
+Improvements
+================================================================================
+
+* :jirabug:`K8SPXC-394`: Support of :ref:`"cluster-wide" mode<install-clusterwide>` for Percona XtraDB Cluster Operator
```

Comment:
I think K8SPXC-394 is a feature, and K8SPXC-144 is an improvement
up to @spron-in

Reply:
I'd say yes, it makes sense to change statuses for these two issues.

Reply:
Yeah, agree.